kotlin-jvm/src/main/sqldelight/util/user.sq

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE user (
+  user_id_pk INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
```

Comment:
Why not simply use `user_id`?

Reply:
I thought it would be better to differentiate primary keys and foreign keys.
Changed it to `user_id`.